squad/README.md

```diff
@@ -0,0 +1,11 @@
+# SQUAD Dedicated Server
+
+## Template OS suport
+
+This template only works for the folling operating systems:

```

Comment:
Typo: "folling"

Reply:
Thanks!! The adjustments has been made 👌